js_modules/dagster-ui/packages/ui-core/src/live-data-provider/util.ts

```diff
@@ -1,13 +1,13 @@
 // How many assets to fetch at once
 const urlParams = new URLSearchParams(window.location.search);
 const liveDataBatchSize = parseInt(urlParams.get('live-data-batch-size') ?? '10', 10);
-const batchThreads = parseInt(urlParams.get('live-data-parallel-fetches') ?? '1', 10);
+const batchThreads = parseInt(urlParams.get('live-data-parallel-fetches') ?? '2', 10);
 
 export const BATCH_SIZE = isNaN(liveDataBatchSize) ? 10 : liveDataBatchSize;
-export const BATCH_PARALLEL_FETCHES = isNaN(batchThreads) ? 1 : batchThreads;
+export const BATCH_PARALLEL_FETCHES = isNaN(batchThreads) ? 2 : batchThreads;
 
 // Milliseconds we wait until sending a batched query
-export const BATCHING_INTERVAL = 250;
+export const BATCHING_INTERVAL = 50;
```

Comment:
whats this timing change about?

Reply:
This is how long we wait before sending out a batch requests. It's to give the UI time to have multiple `useLiveData` calls be batched into a single request even if they occur in different rendering frames. I decreased it shave 200ms off of our call since these queries are on the critical path.

Reply:
(In practice I'm not even sure this is batching much, in retrospect I wish I just used requestAnimationFrame)

Reply:
its not very difficult, I'll put up a follow-up pr to change this